extension/lib/shopgate/CartExtensionPipeline.js

```diff
@@ -86,6 +86,14 @@ class ShopgateCartExtensionPipeline {
     return this._bigCommerceCartRepository.getCheckoutUrl()
   }
 
+  /**
+   * @param {[string]} cartItemIds
```

Comment:
The way we document arrays is for example `string[]`.

Reply:
corrected